src/Stackctl/Spec/Changes.hs

```diff
@@ -70,4 +72,7 @@ runChanges ChangesOptions {..} = do
           let
             name = pack $ stackSpecPathFilePath $ stackSpecSpecPath spec
             formatted = formatChangeSet colors name scoFormat mChangeSet
-          liftIO $ T.appendFile scoOutput $ formatted <> "\n"
+
+          case scoOutput of
+            Nothing -> pushLogger formatted
+            Just p -> liftIO $ T.appendFile p $ formatted <> "\n"
```

Comment:
I just realized this will dump color escapes into the file if stdout is connected to a terminal, since we're just using `getColorsStdout` unrelated to the PATH argument. I won't address it in this PR, but will follow up.